pkg/tasks/python.go

```diff
@@ -14,6 +15,7 @@ func init() {
 	allTasks["python"] = newPython
```

Comment:
Do we still need that?

Reply:
What exactly? The `allTasks` map?

This PR doesn't change how we build the task from the yaml manifest.